pkg/logql/sketch/cms.go

```diff
@@ -19,10 +19,10 @@ func NewCountMinSketch(w, d uint32) (*CountMinSketch, error) {
 	}, nil
 }
 
-func make2dslice(col, row uint32) [][]uint32 {
-	ret := make([][]uint32, row)
+func make2dslice(col, row uint32) [][]float64 {
+	ret := make([][]float64, row)
```

Comment:
we'll have to reevaluate the CMS sizing and overall memory usage with this change, it essentially doubles the memory size of the sketch